lib/tasks/access.rake

```diff
@@ -0,0 +1,9 @@
+require_all 'lib/access_import'
+
+namespace :access do
+  desc 'TODO'
+  task import: :environment do
+    @acimport = AccessImport.new('db/volunteers_data.accdb')
```

Comment:
yes. definiteley! Not only for keeping git clean, this data is also private, so should never ever be in our git repo.